src/algebra/order/floor.lean

```diff
@@ -81,6 +81,12 @@ def ceil : α → ℕ := floor_semiring.ceil
 notation `⌊` a `⌋₊` := nat.floor a
 notation `⌈` a `⌉₊` := nat.ceil a
 
+lemma exists_nat_ge (x : α) : ∃ (n : ℕ), x ≤ ↑n :=
+⟨⌈x⌉₊, (floor_semiring.gc_ceil x ⌈x⌉₊).mp rfl.le⟩
+
+lemma exists_nat_gt [nontrivial α] (x : α) : ∃ (n : ℕ), x < ↑n :=
+⟨_, lt_of_le_of_lt ((floor_semiring.gc_ceil x ⌈x⌉₊).mp rfl.le) (cast_lt.mpr (lt_succ_self ⌈x⌉₊))⟩
```

Comment:
```suggestion
⟨⌈x⌉₊ + 1, (le_ceil x).trans_lt $ strict_mono_cast $ lt_succ_self _⟩
```
which again belongs below `le_ceil`

Reply:
`strict_mono_cast` _definitely_ requires a linear order, which is what I'm hoping to avoid here

Reply:
[citation needed](https://leanprover-community.github.io/mathlib_docs/data/nat/cast.html#nat.strict_mono_cast)

Reply:
Oh sorry! I was confusing it with a different strict monotone. Stared at this too long last night!